Sources/Decorator/Extensions+HKElectrocardiogram.swift

```diff
@@ -62,6 +59,29 @@ extension HKElectrocardiogram.VoltageMeasurement: Harmonizable {
 // MARK: - CustomStringConvertible
 @available(iOS 14.0, *)
 extension HKElectrocardiogram.Classification: CustomStringConvertible {
+    public var key: String {
```

Comment:
I would rather not introduce another field for the JSON payload, but change the values in description [here](https://github.com/VictorKachalov/HealthKitReporter/pull/16/files#diff-9bce61602cbe6016aa7e2e3eecedd0f50aa9ba8a68a625f4ba783e225ca2742eR85)